src/Sensors/GPS/GPS.h

```diff
@@ -1,46 +1,47 @@
 #ifndef GPS_H
 #define GPS_H
 
-#include <imumaths.h>
 #include "../Sensor.h"
 #include "../../Constants.h"
+#include "../../Math/Vector.h"
 
 namespace mmfs
 {
     class GPS : public Sensor
     {
     public:
         virtual ~GPS(){};
-        virtual double getAlt() const;
-        virtual imu::Vector<3> getVelocity() const;
```

Comment:
Even though (if I'm remembering right) the GPS velocity wasn't completely accurate, do we want to completely remove the ability to get velocity from the GPS?

Reply:
The problem here was not that it was inaccurate, but that the GPS sensor was never providing it, and so it doesn't really belong here. The velocity was being calculated by us manually. Not some internal sensor algorithm. If a user wants specifically the GPS velocity, I believe it was decided that it would be better for them to do it themselves in State.

Reply:
That works for me